kalite/project/settings/screenshots.py

```diff
@@ -0,0 +1,27 @@
+"""
+This module is for running a screenshot server (internal FLE purpose mainly).
+To use it, run kalite with --settings=kalite.project.settings.screenshots or
+set then environment variable DJANGO_SETTINGS_MODULE to
+'kalite.project.settings.screenshots'.
+"""
```

Comment:
Yes, --settings is picked up by the basic django management execution stuff.. it overwrites DJANGO_SETTINGS_MODULE